access-grant/src/main/java/com/inrupt/client/accessgrant/AccessGrantClient.java

```diff
@@ -925,7 +917,7 @@ static boolean isAccessDenial(final URI type) {
     /**
      * A data object for verification responses.
      */
-    static class VerificationResponseData {
+    public static class VerificationResponseData {
```

Comment:
If we are going to make this public, then I'd add proper getters and setters to the class